apps/auth_zero/migrations/0004_auto_20201218_1031.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.0.7 on 2020-12-18 10:31
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth_zero", "0003_auto_20201211_1219"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="last_otp",
+            field=models.CharField(max_length=20, null=True),
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="mobile",
+            field=models.CharField(
```

Comment:
install  https://pypi.org/project/Babel/ and use [phonenumbers](https://github.com/stefanfoulis/django-phonenumber-field) which is already installed in the project.
add PHONENUMBER_DEFAULT_REGION to `.env`